test/gradle/executor-smoke/crd-workflow/smoke.yaml

```
@@ -61,3 +61,32 @@ spec:
   events:
   - cronjob:
       cron: "10 */4 * * *"
+---
+apiVersion: testworkflows.testkube.io/v1
+kind: TestWorkflow
+metadata:
+  name: gradle-workflow-smoke-jdk11-from-template
+  labels:
+    core-tests: workflows
+spec:
+  content:
+    git:
+      uri: https://github.com/kubeshop/testkube
+      revision: main
+      paths:
+      - contrib/executor/gradle/examples/hello-gradle
+  container:
+    resources:
+      requests:
+        cpu: 512m
+        memory: 512Mi
+    workingDir: /data/repo/contrib/executor/gradle/examples/hello-gradle
+    env:
+      - name: TESTKUBE_GRADLE
+        value: "true"
```

Guess that it's more common in our codebase to avoid indent for lists:
```suggestion
    - name: TESTKUBE_GRADLE
      value: "true"
```